Index: src/model/Graph.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/model/Graph.java b/src/model/Graph.java
--- a/src/model/Graph.java	
+++ b/src/model/Graph.java	
@@ -56,6 +56,7 @@
 
         /*Algoritmo de Dijkstra*/
         PriorityQueue<Node<T>> priorityQueue = new PriorityQueue<>();
+        ArrayList<T> path = new ArrayList<>();
         priorityQueue.add(new Node<>(origin, .0));
         while(!priorityQueue.isEmpty()) {
             double distance = -priorityQueue.peek().weight;
@@ -66,6 +67,7 @@
                 T child = node.key;
                 if(distances.get(child) > distances.get(root) + weight) {
                     distances.put(child, distances.get(root) + weight);
+                    path.add(node.key);
                     priorityQueue.add(new Node<>(child, -distances.get(child)));
                 }
             }
